doc/nrf/ug_gzll.rst

```diff
@@ -79,10 +72,8 @@ See the :ref:`nrfxlib:gzll_api` for a list of configuration functions as well as
 Gazell modes
```

Comment:
Essentially the Gazell protocol is asymmetrical and has 2 modes: host (listening node) and device (transmitting node). It can run in mode at a time, and the mode can be changed in runtime. I don't know if anything more can be written here.

Reply:
Okay, I will add this. Thank you @grochu!

Reply:
The statement "Each Device communicates to a single Host" appears in the star network paragraph. As shown in the figure, it is valid.

The statement "a single Device can speak to several Hosts" appears in the "more sophisticated Gazell networks" paragraph. Such a configuration is not a star network. This scenario is a possibility rather than a common usage. Supporting 2 ESB base addresses, a Gazell Device can speak to at least 2 Hosts since nRF24L01 days. For 3 or most Hosts, some Hosts need to use the same base address and different prefix byte. It boils down to the MultiCeiver design in nRF24L01.

The nRF5 SDK GZLL user guide puts the focus on star network. It is up to the interested user to explore other possibilities.

I agree that the 'Gazell modes' is confusing. In nRFGo SDK, they call it 'roles'. See following section:

**Roles in a Gazell star network**
In a simple Gazell star network we have two static roles: Host and Device.

The term 'mode' may come from the GZLL API. There the user input a 'mode' to nrf_gzll_init() function to select the 'role'.

https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrfxlib/gzll/doc/api.html#c.nrf_gzll_mode_t

How about modifying the paragraph header to 'Gazell network'?

Reply:
Thanks @leewkb4567 for the comprehensive response. Then I agree the statements I questioned are OK as long as they are used within the context of a subsection (the "star network" part and the "more sophisticated... part respectively").
I agree "_role_" is probably a better term than "_mode_"

Reply:
I agree, and have made this change throughout the documentation.